### ISSUE TITLE
Date dependent listing

### ISSUE THREAD

**Issue by robtarr** [gemcityjs/gemcityjs#2]
The https://github.com/gemcityjs/gemcityjs/blob/master/templates/pages/index.hbs doesn't automatically show the right meeting based on date. It currently needs the `when` field set to `past`, `upcoming`, or `next`.

We would like this to be automated solely on the date and this property removed.

**Comment by benhastings** [comment]
To clarify: You're looking to eliminate the need for the "when" field in the meetings.yml file and populate the "next meeting" field based on the "date" field. Correct?

**Comment by moritzsternemann** [comment]
Not only the "next meeting", but "upcoming" and "past" as well, right?

**Comment by robtarr** [comment]
Correct.

**Comment by robtarr** [comment]
Yes, we would like the `when` field completely removed, and have the entries show up in the correct places solely based on the listed date.

**Comment by moritzsternemann** [comment]
Okay, I would like to work on it if @benhastings doesn't claim it.. 😄

**Comment by benhastings** [comment]
Already have a solution. Just didn't get to push it up yet.
